docs/source/howto/installation.rst

```diff
@@ -223,24 +223,344 @@ the configuration directory ``~/project_a/.aiida`` will be used.
+
+Updating from 0.x.* to 1.*
+--------------------------
+- `Additional instructions on how to migrate from 0.12.x versions <https://aiida.readthedocs.io/projects/aiida-core/en/v1.2.1/install/updating_installation.html#updating-from-0-12-to-1>`_.
+- `Additional instructions on how to migrate from versions 0.4 -- 0.11 <https://aiida.readthedocs.io/projects/aiida-core/en/v1.2.1/install/updating_installation.html#older-versions>`_.
+- For a list of breaking changes between the 0.x and the 1.x series of AiiDA, check `this page <https://aiida.readthedocs.io/projects/aiida-core/en/v1.2.1/install/updating_installation.html#breaking-changes-from-0-12-to-1>`_.
 
 
 .. _how-to:installation:backup:
 
 Backing up your installation
 ============================
 
-`#4004`_
+A full backup of an AiiDA instance and AiiDA managed data requires a backup of:
+
+* the profile configuration in the ``config.json`` file located in the ``.aiida`` folder.
+  Typically located at ``~/.aiida`` (see also :ref:`configure_aiida`).
+
+* files associated with nodes in the repository folder (one per profile). Typically located in the ``.aiida`` folder.
+
+* queryable metadata in the PostgreSQL database (one per profile).
+
+
+.. _how-to:installation:backup:repository:
+
+Repository backup (``.aiida`` folder)
+-------------------------------------
+
+For **small repositories** (with less than ~100k files), simply back up the ``.aiida`` folder using standard backup software.
+For example, the ``rsync`` utility supports incremental backups, and a backup command might look like ``rsync -avHzPx`` (verbose) or ``rsync -aHzx``.
+
+For **large repositories** with millions of files, even incremental backups can take a significant amount of time.
+AiiDA provides a helper script that takes advantage of the AiiDA database in order to figure out which files have been added since your last backup.
+The instructions below explain how to use it:
+
+ 1. Configure your backup using ``verdi -p PROFILENAME devel configure-backup`` where ``PROFILENAME`` is the name of the AiiDA profile that should be backed up.
+    This will ask for information on:
+
+    * The "backup folder", where the backup *configuration file* will be placed.
+      This defaults to a folder named ``backup_PROFILENAME`` in your ``.aiida`` directory.
+
+    * The "destination folder", where the files of the backup will be stored.
+      This defaults to a subfolder of the backup folder but we **strongly suggest to back up to a different drive** (see note below).
+
+    The configuration step creates two files in the "backup folder": a ``backup_info.json`` configuration file (can also be edited manually) and a ``start_backup.py`` script.
+
+    .. dropdown:: Notes on using a SSH mount for the backups (on Linux)
```

Comment:
Again, at least on my local computer, this doesn't work.

Reply:
See above